crates/stake-chain/src/lib.rs

```diff
@@ -0,0 +1,29 @@
+//! # Strata Bridge Stake Chain
+//!
+//! In a BitVM-based brideg, each transaction graph requires a stake that gets burnt/slashed/spent
```

Comment:
```suggestion
//! In a BitVM-based bridge, each transaction graph requires a stake that gets burnt/slashed/spent
```
Should have been caught by `codespell` 🤦

Reply:
Done in e92bf9e08ca66ef6decd1615ad132d3e324e27cc
Thanks!